### PR TITLE
UI-III-Flexbox -- Aaron Gandee 

### PR DESCRIPTION
**Pull request by AaronGandee**
***This is still far from done***, i attempted to start over from scratch and messed everything up and ran out of time. i will be working on fixing and correcting the rest of it and will have it done by the end of the evening. sorry its not finished i'm just having a very hard time with time management and i tend to panic when i get lost and am on the clock.  i will get it solved by tonight though.  Im just having an embarrassingly hard time at them moment.

### CODE REVIEW

**Comment by AaronGandee** [comment]
I have finished working on the project, it is as close as i can manage with out further practice.  its not 100% awesome but i tried very hard for hours, so i'm pretty happy with it.  please review and thank you for your time.

File: great-idea/index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+!doctype html>
```
Comment (josenriagu) [inline great-idea/index.html]: You missed out the < tag and this broke your website!

File: great-idea/css/index.css [hunk 2]
```diff
@@ -63,4 +63,210 @@ h1, h2, h3, h4, h5 {
...
+}
+ 
+p {
+	position: relative; 
+	bottom: 72px;
+	left: 10px
+}
+/* css for services tab */
+
+.servimghead {
```
Comment (josenriagu) [inline great-idea/css/index.css]: Hey buddy! You could do all of the project with **display: flex** and flex properties without **position**. Position will make it tougher and may mess things up

File: great-idea/services.html [hunk 3]
```diff
@@ -13,95 +13,106 @@
...
-Awesome thing
-Amazing thing
-Cool thing
-Great thing
-
-<img class="services-info-img" src="img/services-info.png" alt="one of our employees hard at work">
-
-Copyright Great Idea! 2018
-
+  <div class="container">
```
Comment (josenriagu) [inline great-idea/services.html]: good job. you could use prettier formatting plugin to make your codes more readable

**Pull request merged**